_posts/blog/2018-01-24-modernizing-mlab.md

```diff
@@ -9,18 +9,21 @@ categories:
   - virtualization
   - kernel
   - tcp-info
+  - bbr
   - web100
 ---
 
 # Modernizing the M-Lab Platform
 {% include post-meta.html %}
 
-When the M-Lab platform was initially launched in 2009, the software and operating system running on our servers used the best available boot management, virtualization, and kernel-level measurement instrumentation available. In the years since M-Lab’s initial launch, the state of system administration has improved dramatically. In 2016, the M-Lab team began work to upgrade the platform to adopt modern and flexible system administration components. This post provides a roadmap of that work.
+When the M-Lab platform was initially launched in 2009, the software and operating system running on our servers used the best available boot management, virtualization, and kernel-level measurement instrumentation available. In the years since M-Lab’s initial launch, the state of system administration has improved dramatically. In 2017, the M-Lab team began work to upgrade the platform to adopt modern and flexible system administration components. This post provides a roadmap of that work.
 <!--more-->
 
 ## M-Lab Now
 
-All M-Lab servers today are hosted either in well-connected data centers or within the research and education networks of contributing partner institutions. Each location has at least one "pod" which consists of 3-4 servers and one switch, connected directly to an upstream transit provider. Each M-Lab pod has between 1-10 Gbps upstream connectivity, except in some regions of the world where the internet is less developed.
+All M-Lab servers today are hosted either in well-connected data centers or within the research and education networks of contributing partner institutions. Each location has at least one "pod" which consists of 3-4 servers and one switch, connected directly to an upstream transit provider. Each M-Lab pod has between 1-10 Gbps upstream connectivity.
+
+The kernel, virtualization, and operating system running on M-Lab servers has historically been based on [PlanetLab](https://www.planet-lab.org/){:target="_blank"}. Since 2003, the PlanetLab Consortium has provided a research platform for researchers to prototype and develop, "new technologies for distributed storage, network mapping, peer-to-peer systems, distributed hash tables, and query processing." During this modernization project, M-Lab will remove most of our technical dependencies on PlanetLab. However, PlanetLab will continue to play an important role in the M-Lab Consortium.
```

Comment:
Can we just delete this paragraph? I keep trying to rewrite it and not coming up with a good rewrite.

Reply:
@pboothe Which paragraph?